flow360/component/simulation/translator/solver_translator.py

```diff
@@ -447,6 +447,48 @@ def porous_media_translator(model: PorousMedium):
     return porous_medium
 
 
+def bet_disk_entity_info_serializer(volume):
+    """BET disk entity serializer"""
+    v = convert_tuples_to_lists(remove_units_in_dict(dump_dict(volume)))
+    return {
+        "axisOfRotation": v["axis"],
+        "centerOfRotation": v["center"],
+        "radius": v["outerRadius"],
+        "thickness": v["height"],
+    }
+
+
+def bet_disk_translator(model: BETDisk):
+    """BET disk translator"""
+    disk_param = convert_tuples_to_lists(remove_units_in_dict(dump_dict(model)))
+    replace_dict_key(disk_param, "machNumbers", "MachNumbers")
+    replace_dict_key(disk_param, "reynoldsNumbers", "ReynoldsNumbers")
+    disk_param.pop("volumes")
```

Comment:
This introduces coupling between BETDisk class and the translated JSON. It is probably fine because BETDisk is huge but we need good unit test.

Reply:
Right, I can decouple them

Reply:
Ok then please do that.